static/js/main.js

```diff
@@ -0,0 +1,129 @@
+
+function preloadArchi()
+{
+    let darkmode = getCookie('darkmode');
+    if(darkmode && darkmode.length > 0)
+    {
+       switch(darkmode)
+       {
+           case "on":
+                switchDarrMode('dark');
+            break;
+            case "off":
+                switchDarrMode('main');
+            break;
+
+       }
+    }
+    else
+    {
+        let mode = getDarkModeInit();
+        switch(mode)
+        {
+            case "dark":
+                    setCookie('darkmode', 'off');
+                break;
+                case "main":
+                case "auto":
+                    setCookie('darkmode', 'on');
+                break;
+
+        }
+    }
+}
+
+function toggleDarkMain()
+{
+    let darkmode = getCookie('darkmode');
+    if(darkmode && darkmode.length > 0)
+    {
+       let mode = getDarkModeInit();
+
+       switch(mode)
+       {
+           case "dark":
+                setCookie('darkmode', 'off');
+                switchDarrMode('main');
+            break;
+            case "main":
+            case "auto":
+                setCookie('darkmode', 'on');
+                switchDarrMode('dark');
+            break;
+
+       }
+    }
+    else
+    {
+        //console.log('no existe');
+        setCookie('darkmode', 'on');
+    }
+}
+
+function getDarkModeInit()
+{
+   let mode = ""; 
+   let likns =  Array.from(document.querySelectorAll('head > link'));
+   if (likns)
+   {
+    likns.map(item => {
+        if(item.media && item.media.includes('prefers-color-scheme'))
+        {
+            mode = item.media.split(":");
+        }
+       });
+   }
+   return (mode) ? (mode.length > 1) ? String(String(mode[1]).split(")")[0]).trim() : null : null;
+}
+
+
+function switchDarrMode(_mode)
+{
+   let likns =  Array.from(document.querySelectorAll('head > link'));
+   if (likns)
+   {
+    likns.map(item => {
+        if(item.media && item.media.includes('prefers-color-scheme'))
+        {
+            item.media = `(prefers-color-scheme: ${_mode})`;
+        }
+       });
+   }
+}
+
+
+// ::: cookies :::
```

Comment:
Not required if switched to localStorage

Reply:
Done 💯